spec/spec_helper.rb

```diff
@@ -19,5 +19,5 @@
   config.cassette_library_dir = "spec/vcr_cassettes"
   config.hook_into :webmock
   config.configure_rspec_metadata!
-  config.filter_sensitive_data("<PINGEN_API_TOKEN>") { ENV["PINGEN_API_TOKEN"] }
+  config.filter_sensitive_data("PINGEN_API_TOKEN") { ENV["PINGEN_API_TOKEN"] }
```

Comment:
It doesn't work with the `<` and `>` because the token is in the URL and those characters are not valid in that context